NeoGeoMiniRomAdder/Form1.Designer.cs

```diff
@@ -118,34 +123,38 @@ partial class Form1 {
             this.flowLayoutPanel2.Controls.Add(this.pbLCD);
             this.flowLayoutPanel2.Controls.Add(this.pbTV);
             this.flowLayoutPanel2.Controls.Add(this.bLoadLCD);
-            this.flowLayoutPanel2.Location = new System.Drawing.Point(359, 3);
+            this.flowLayoutPanel2.Location = new System.Drawing.Point(420, 3);
+            this.flowLayoutPanel2.Margin = new System.Windows.Forms.Padding(4, 3, 4, 3);
             this.flowLayoutPanel2.Name = "flowLayoutPanel2";
-            this.flowLayoutPanel2.Size = new System.Drawing.Size(395, 275);
+            this.flowLayoutPanel2.Size = new System.Drawing.Size(461, 317);
             this.flowLayoutPanel2.TabIndex = 7;
             // 
             // pbLCD
             // 
             this.pbLCD.BackColor = System.Drawing.SystemColors.ControlDark;
-            this.pbLCD.Location = new System.Drawing.Point(3, 3);
+            this.pbLCD.Location = new System.Drawing.Point(4, 3);
+            this.pbLCD.Margin = new System.Windows.Forms.Padding(4, 3, 4, 3);
             this.pbLCD.Name = "pbLCD";
-            this.pbLCD.Size = new System.Drawing.Size(94, 78);
```

Comment:
Please restore pbLCD to its original size to preserve accuracy to the actual LCD image size